frontend/src/components/pages/HomePage/HomePage.tsx

```diff
@@ -55,15 +55,15 @@ const HomePage = (): React.ReactElement => {
   };
 
   const getLogRecords = async (pageNumber: number) => {
-    const buildingValue = building ? building.value : "";
+    const buildingId = building ? building.value : undefined;
```

Comment:
should use `null` instead here - it's a better representation of absence of a value, whereas `undefined` should be used for when you've defined a variable as empty and are going to assign to it later